Documentation/minimum_requirements.md

```diff
@@ -0,0 +1,14 @@
+#.NET Core Minimum System Requirements:
+
+In general, the minimum requirements of your operating system will be enough to run .NET Core applications.
+
+##1.X 
+ 
+The minimum available resources required for a simple standalone application are:
+ 
+- 25MB Disk Space
+- 16MB available RAM
+
+Depending on your application and performance requirements, these requirements may be higher. 
+
+*SSE2 processor instruction support is required.
```

Comment:
This is only relevant for x86/64 processors, though. Should there be a list of architectures supported, instead, and put this as a sub-bullet under x86-64?